src/Plugins/ComplexCore/docs/ResampleImageGeomFilter.md

```diff
@@ -6,19 +6,35 @@ Sampling (Resample)
 
 ## Description ##
 
-This **Filter** changes the **Cell** spacing/resolution based on inputs from the user. The values entered are the desired new spacings (not multiples of the current resolution).  The number of **Cells** in the volume will change when the spacing values are changed and thus the user should be cautious of generating "too many" **Cells** by entering very small values (i.e., very high resolution). Thus, this **Filter** will perform a down-sampling or up-sampling procedure.  
+This **Filter** changes the **Cell** spacing/resolution based on inputs from the user. There are several resampling modes:
+
+### Spacing ###
+The values entered are the desired new spacings (not multiples of the current resolution).  The number of **Cells** in the volume will change when the spacing values are changed and thus the user should be cautious of generating "too many" **Cells** by entering very small values (i.e., very high resolution).
+
+### Scale Factor ###
+The values entered are the desired scaling factor for each dimension, in percentages.  (50%, 50%, 50%) would resample the geometry by half in all dimensions.
```

Comment:
Can you give a more concrete example by stating for a given scaling and dimensions, if I put in 50% in all directions, what do I end up with for dimensions and scaling.